nixos/modules/services/databases/postgresql.nix

```diff
@@ -88,6 +90,16 @@ in
         '';
       };
 
+      initdbArgs = mkOption {
+        type = with types; listOf str;
+        default = [];
+        example = [ "--data-checksums" "--allow-group-access" ];
+        description = ''
+          Additional arguments passed to <literal>initdb<literal> during data dir
```

Comment:
For the record, this breaks the manual build: 6c6310787212acaba1b8f3a9b3bfd319015f00e2

Reply:
:-( thanks for the fix